src/query/mod.rs

```diff
@@ -23,3 +23,6 @@ pub use update::*;
 pub use util::*;
 
 pub use crate::{ConnectionTrait, InsertResult, Statement, UpdateResult, Value, Values};
+
+// Imports paginator utility by default making it backward compatible.
+pub use crate::PaginatorTrait;
```

Comment:
Not nitpicking, but it actually can fit in the above statement.

Reply:
Can ah, just wanna put an explicit comment on it. I can remove the comment if it's unnecessary

Reply:
It's unnecessary.